### PR TITLE
chore(deviceauth cronjob): to run the license count

### PR DESCRIPTION
**Pull request by 0lmi**
Changelog: Title
Ticket: MEN-5898
Signed-off-by: Alex Miliukov <oleksandr.miliukov@northern.tech>

### CODE REVIEW

File: mender/templates/device-auth-cron-license-count.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+            envFrom:
+            - prefix: DEVICEAUTH_
+              secretRef:
+                name: mongodb-common
+
+          volumes:
+          - name: rsa
+            secret:
+              secretName: rsa-device-auth
```
Comment (alfrunes) [inline mender/templates/device-auth-cron-license-count.yaml:46]: The command does not require this secret.

File: mender/templates/device-auth-cron-license-count.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+          containers:
+          - name: device-auth-license-count
+            image: {{ .Values.device_auth.image.registry | default "registry.mender.io" }}/{{ .Values.device_auth.image.repository | default "mendersoftware/deviceauth-enterprise" }}:{{ .Values.device_auth.image.tag }}
+            command: ["/usr/bin/deviceauth-enterprise"]
+            args: ["license-count"]
+
+            volumeMounts:
+            - name: rsa
+              mountPath: "/etc/deviceauth-enterprise/rsa/"
+              readOnly: true
```
Comment (alfrunes) [inline mender/templates/device-auth-cron-license-count.yaml:36]: :point_up:

File: mender/templates/device-auth-cron-license-count.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+          {{- with .Values.device_auth.podAnnotations }}
+          annotations:
+            {{- toYaml . | nindent 8 }}
+          {{- end }}
+          labels:
+            app.kubernetes.io/name: device-auth-license-count
+
+        spec:
+          {{- with .Values.device_auth.affinity }}
+          affinity: {{ tpl (toYaml .) $ | nindent 8 }}
```
Comment (alfrunes) [inline mender/templates/device-auth-cron-license-count.yaml:25]: I don't understand the use of `tpl` here, why not
```yaml
          affinity: {{ toYaml . | nindent 8 }}
```
?

File: mender/templates/device-auth-cron-license-count.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+        spec:
+          {{- with .Values.device_auth.affinity }}
+          affinity: {{ toYaml . | nindent 8 }}
+          {{- end }}
+          containers:
+          - name: device-auth-license-count
+            image: {{ .Values.device_auth.image.registry | default "registry.mender.io" }}/{{ .Values.device_auth.image.repository | default "mendersoftware/deviceauth-enterprise" }}:{{ .Values.device_auth.image.tag }}
+            command: ["/usr/bin/deviceauth-enterprise"]
+            args: ["license-count"]
```
Comment (tranchitella) [inline mender/templates/device-auth-cron-license-count.yaml:31]:
```suggestion
            command: ["/usr/bin/deviceauth-enterprise", "license-count"]
```

**Review by tranchitella — APPROVED** [review]
Looks good, only a minor suggestion.

**Pull request merged**